app/models/spree/variant.rb

```diff
@@ -38,7 +38,7 @@ class Variant < ApplicationRecord
 
     has_many :stock_items, dependent: :destroy, inverse_of: :variant
     has_many :stock_locations, through: :stock_items
-    has_many :stock_movements, dependent: :destroy
+    has_many :stock_movements, through: :stock_items, dependent: :destroy
```

Comment:
Was this broken before? That's interesting, perhaps we weren't using it and should consider removing it instead..

Reply:
Yeah, the destroy wasn't working without this `through:`. A [quick search](https://github.com/search?q=repo%3Aopenfoodfoundation%2Fopenfoodnetwork%20stock_movements&type=code) seems to reveal that this relation is never used, it's only used with `stock_item`.

Should I just delete it?

Reply:
Thanks, yes it sounds like it!
FYI @mkllnk

Reply:
I think we agreed to delete association entirely, as it's unused. The cleanup  is done by the `has_many :stock_items` association, so we don't need to specify anything here.
```suggestion
```